k9mail/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -160,6 +183,18 @@ public boolean onTouchEvent(@NonNull MotionEvent event) {
                 TokenImageSpan[] links = text.getSpans(offset, offset, RecipientTokenSpan.class);
                 if (links.length > 0) {
                     showAlternates(links[0].getToken());
+                    handler.removeCallbacks(recipientLongPressed);
+                    return true;
+                }
+            }
+        }
+        else if(text != null && event.getAction() == MotionEvent.ACTION_DOWN) {
+            int offset = getOffsetForPosition(event.getX(), event.getY());
+            if (offset != -1) {
+                TokenImageSpan[] links = text.getSpans(offset, offset, RecipientTokenSpan.class);
+                if (links.length > 0) {
+                    recipientLongPressed = new LongPressCopy(links[0].getToken().address.getAddress());
+                    handler.postDelayed(recipientLongPressed, 1000);
```

Comment:
1000 is a magic number here (and I'm not sure how it was chosen). I think we probably want to use `android.view.ViewConfiguration.get(this).getLongPressTimeout()` instead so it's consistent with everything else.